src/app/components/ButtonLink/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { Button, ButtonExtendedProps } from 'grommet'
+import { Link, LinkProps } from 'react-router-dom'
+import React from 'react'
+
+type ButtonLinkProps = LinkProps & ButtonExtendedProps
+
+/**
+ * Behaves like {@link Link} and styled like {@link Button}. Needed because:
+ * - Link isn't styled but routes correctly
+ * - Button is styled but can't navigate
+ * - Wrapping Link around Button is bad for accessibility
+ */
+export const ButtonLink = (props: ButtonLinkProps) => {
+  return <Button as={Link} {...props} />
```

Comment:
This throws tons of console warnings, because props are assigned as HTML attributes. HTML markup is also wrong `theme="[object Object]"`